src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -9317,6 +9317,96 @@ private void OnChanged(object sender, NotifyCollectionChangedEventArgs e)
     }
 
     private void OnChanged() { }
+}");
+        }
+
+        [Fact, WorkItem(29761, "https://github.com/dotnet/roslyn/issues/29761")]
+        public async Task GenerateAlternativeNamesForFuncActionDelegates1()
```

Comment:
💡 It would be good to include a test for a delegate type which is not `Action` or `Func` in order to show that the generated parameter name is derived from the corresponding parameter name in the delegate.